apps/api/v2/src/modules/webhooks/webhooks.repository.ts

```diff
@@ -0,0 +1,78 @@
+import { PrismaReadService } from "@/modules/prisma/prisma-read.service";
+import { Injectable } from "@nestjs/common";
+import { v4 as uuidv4 } from "uuid";
+
+import { PrismaWriteService } from "../prisma/prisma-write.service";
+import { CreateWebhookInputDto } from "./inputs/create-webhook.input";
+
+@Injectable()
+export class WebhooksRepository {
+  constructor(private readonly dbRead: PrismaReadService, private readonly dbWrite: PrismaWriteService) {}
+
+  async createUserWebhook(userId: number, data: CreateWebhookInputDto) {
```

Comment:
I have observed this in other repositories - I think that repositories ideally should not know about `CreateWebhookInputDto` or other DTOs - for data we can use `Prisma.xyz` like in :
```
async createTeamEventType(data: Prisma.EventTypeCreateInput) {
    return this.prismaWriteClient.eventType.create({ data });
  }
```

for repository to know just about database stuff.